<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Приложение</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="baseSize">
    <size>
     <width>650</width>
     <height>648</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>CrossCursor</cursorShape>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="save"/>
    <addaction name="open"/>
    <addaction name="loadBackground"/>
    <addaction name="loadcustoBack"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Представить в</string>
    </property>
    <addaction name="action_3D"/>
    <addaction name="action_3D_2"/>
    <addaction name="action_2D"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="menu"/>
  </widget>
  <action name="save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="action_3D">
   <property name="text">
    <string>Представить в 3D как поверхность</string>
   </property>
  </action>
  <action name="action_2D">
   <property name="text">
    <string>Перейти в 2D редактор</string>
   </property>
  </action>
  <action name="loadBackground">
   <property name="text">
    <string>Загрузить сетку</string>
   </property>
  </action>
  <action name="loadcustoBack">
   <property name="text">
    <string>Загрузить замену сетки</string>
   </property>
  </action>
  <action name="action_3D_2">
   <property name="text">
    <string>Представить в 3D как корпус</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWnd.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
